src/i18n/hi/nav.ts

```diff
@@ -1,11 +1,74 @@
 import { NavDictionary } from '../translation-checkers';
 
 export default NavDictionary({
-	// Start Here
 	startHere: 'यहाँ से शुरू करे',
 	'getting-started': 'पहले कदम',
+	install: 'स्थापित करें',
+	'editor-setup': 'संपादक सेटअप',
```

Comment:
`संपादक` here sounds more like a person (ex: newspaper editor). I find `एडिटर` more suitable in this case
```suggestion
	'editor-setup': 'एडिटर सेटअप',
```

Reply:
so should we leave the words which sounds complicated or uncommon?